### PR TITLE
Makes mathjax script optional with config param.

### PR DESCRIPTION
**Pull request by udit-001**

### CODE REVIEW

**Comment by xiaoheiAh** [comment]
please sync config to [config.yml](https://github.com/xiaoheiAh/hugo-theme-pure/blob/master/exampleSite/config.yml) and add comment

**Comment by udit-001** [comment]
I don't know how to do, I'm just learning along as I go, so could you tell me what is it that I need to configure?

File: layouts/partials/script.html [hunk 1]
```diff
@@ -1,12 +1,6 @@
-<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_SVG"></script>
-<script type="text/x-mathjax-config">
-    MathJax.Hub.Config({
-            showMathMenu: false, //disables context menu
-            tex2jax: {
-            inlineMath: [ ['$','$'], ['\\(','\\)'] ]
-           }
-    });
-</script>
+{{ if or .Params.math .Site.Params.math }}
```
Comment (xiaoheiAh) [inline layouts/partials/script.html]: just add this param to config.yml as example to users. 
Parameter names need to have semantics.like this:
```yml
params:
  enableMathJax: true
```

Comment (udit-001) [reply layouts/partials/script.html]: Site wide param is named `enableMathJax`, for a single post they can set `math: true`, you might wanna add info about this in README.

**Comment by xiaoheiAh** [comment]
LGTM

**Pull request merged**